### PR TITLE
Camel Class (Matthew Chan)

### PR DESCRIPTION
**Pull request by mchan7841**
Adding a camel class and a test file for the camel class.

### CODE REVIEW

**Review by Skaurm — COMMENTED** [review]
Missing documentation for Camel

**Review by Skaurm — APPROVED** [review]
Looks good, ready to merge

**Review by shayaanmk — APPROVED** [review]
Ready to merge

File: src/main/java/Trader.java [hunk 1]
```diff
@@ -100,6 +100,7 @@ public static void main(String[] args) {
                 new Camel(),
                 new Camel(),
                 new Camel()
```
Comment (Skaurm) [inline src/main/java/Trader.java]: Comma missing

Comment (shayaanmk) [inline src/main/java/Trader.java]: Add comma

**Review by shayaanmk — CHANGES_REQUESTED** [review]

**Review by Skaurm — CHANGES_REQUESTED** [review]
Comma missing

**Review by shayaanmk — APPROVED** [review]
Good to merge

**Review by Skaurm — APPROVED** [review]
approved

**Pull request merged**